.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
```

Comment:
Why did you remove these lines? FOSUserBundle 1.3 dies work with PHP 5.3 and so tests should be executed on this version too.